webview-ui/.storybook/main.ts

```diff
@@ -0,0 +1,16 @@
+import type { StorybookConfig } from "@storybook/react-vite"
+
+const config: StorybookConfig = {
+	stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
+	addons: [
```

Comment:
Not sure which of these are actually needed; this was auto-generated by Storybook.